<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerWindow</class>
 <widget class="QMainWindow" name="WorkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worker Terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1e1e1e;
    color: #ffffff;
}
QLineEdit {
    background-color: #2d2d2d;
    border: 1px solid #3e3e3e;
    color: #ffffff;
    padding: 5px;
    border-radius: 4px;
}
QPushButton {
    background-color: #007acc;
    color: white;
    border: none;
    padding: 8px;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #0098ff;
}
QPushButton:disabled {
    background-color: #333333;
    color: #888888;
}
QListWidget {
    background-color: #252526;
    border: 1px solid #3e3e3e;
    color: #cccccc;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MY IDEAS:</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold; font-size: 14px; color: #007acc;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listMyIdeas"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditIdea">
        <property name="placeholderText">
         <string>Enter idea...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>